src/plugins/kibana/public/discover/styles/main.less

```diff
@@ -70,13 +77,15 @@
     }
   }
 
+  &-actions {
```

Comment:
See above, the styling for the navbar elements should be moved out into a shared `.less` since these sorts of buttons and titles are reused across many applications